website/orders/services.py

```diff
@@ -131,7 +135,10 @@ def add_user_order(product: Product, shift: Shift, user: User) -> Order:
         type=Order.TYPE_ORDERED,
         user=user,
         user_association=user.association,
+        paid=paid,
+        ready=ready,
         prioritize=user_gets_prioritized_orders(user, shift),
+        deprioritize=deprioritize,
     )
 
 
```

Comment:
Can some of the setters below be removed now?

Reply:
Why?

Reply:
Because they're no longer used (e.g. set_shift_active, set_order_ready), so its clean to not leave them behind. Also since the API can now change the models without going through the services, leaving them could falsely suggest that any logic put in them would be adhered to, whereas it will actually never run.

Reply:
Ahh I was looking at other pieces of code, I removed them!